pkg/runtime/interface.go

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime
+
+import (
+	"net"
+
+	v2 "github.com/sealerio/sealer/types/api/v2"
+)
+
+type Interface interface {
+	// Init exec kubeadm init
```

Comment:
should be deleted?

Reply:
yep! it will not just for kubeadm.

Reply:
You must add more annotation for interface functions including function meaning, input parameters, returned parameter. This is quite important, since it is interface.

Reply:
Fine, i will add annotation for every interface function.